tests/lib/rules/no-private-routing-service.js

```diff
@@ -38,6 +38,9 @@ ruleTester.run('no-private-routing-service', rule, {
       }
     }
     `,
```

Comment:
Can you add a test case to make sure native classes handle the decorator with no arguments?
```
class MyComponent extends Component { @service() routing; }
```

Reply:
yes good call!